services/movie.py

```diff
@@ -0,0 +1,42 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
```

Comment:
According to the checklist, you should not load the database when it's not needed. In this case, you are loading all movies from the database with `Movie.objects.all()`, even if you are going to filter them later. It would be more efficient to start building your queryset based on the provided filters (genres_ids and actors_ids) right away, without loading all movies first.